controller/QtiCreator.php

```diff
@@ -300,6 +300,10 @@ protected function getCreatorConfig(core_kernel_classes_Resource $item)
             $config->addPlugin('blockAdder', 'taoQtiItem/qtiCreator/plugins/content/blockAdder', 'content');
         }
 
+        if ($config->getProperty('scrollable-multi-column') == true) {
```

Comment:
Should we apply here strict comparison?
```suggestion
        if ($config->getProperty('scrollable-multi-column') === true) {
```